tests/integration/test_trading.py

```diff
@@ -294,3 +299,795 @@ def test_estimate_position(vega_service_with_market: VegaServiceNull):
         is_market_order=[False, False],
         collateral_available=1,
     )
+
+@pytest.mark.integration
+def test_liquidation_and_estimate_position_calculation_AC001003(vega_service: VegaServiceNull):
+    vega = vega_service
+    WalletConfig = namedtuple("WalletConfig", ["name", "passphrase"])
+
+    MM_WALLET = WalletConfig("mm", "pin")
+    PARTY_A = WalletConfig("party_a", "party_a")
+    PARTY_B = WalletConfig("party_b", "party_b")
+    PARTY_C = WalletConfig("party_c", "party_c")
+    TERMINATE_WALLET = WalletConfig("TERMINATE", "TERMINATE")
+    LIQ_WALLET = WalletConfig("LIQ", "TRADER")
+    ASSET_NAME = "tDAI"
+    WALLETS = [MM_WALLET, PARTY_B, PARTY_C, TERMINATE_WALLET, LIQ_WALLET]
+
+    mint_amount = 100000000
+    initial_volume = 10
+    initial_commitment = 1500
+    collateral_available = 252
+
+    vega.wait_for_total_catchup()
+    for wallet in WALLETS:
+        vega.create_key(wallet.name)
+    vega.create_key(PARTY_A.name)
+
+    vega.wait_for_total_catchup()
+    vega.mint(
+        MM_WALLET.name,
+        asset="VOTE",
+        amount=1e4,
+    )
+    vega.forward("10s")
+    vega.wait_for_total_catchup()
+    vega.wait_fn(1)
+
+    configWithSlippage = MarketConfig()
+    configWithSlippage.load()  # Load the default configuration
+    configWithSlippage.set("linear_slippage_factor", str(1))  # Set the linear_slippage_factor to 0.5
+    configWithSlippage.set("quadratic_slippage_factor", str(1))  # Set the quadratic_slippage_factor to 0.5
+    configWithSlippage.set("decimal_places", int(0))  # Set the market decimal_places to 0
+    configWithSlippage.set("lp_price_range", str(1))
+
+    # Initialize the Market Manager
+    marketManager = ConfigurableMarketManager(
+        proposal_key_name="MM_WALLET.name",
```

Comment:
This works but just to be clear this is setting the key to the string `"MM_WALLET.name"` rather than the value of `MM_WALLET.name` which is `"mm"`. Might be better off using `MM_WALLET.name`.